src/fastcs_odin/eiger_fan.py

```diff
@@ -1,9 +1,18 @@
-from fastcs_odin.odin_adapter_controller import OdinAdapterController
+from fastcs_odin.odin_subcontroller import OdinSubController
+from fastcs_odin.util import create_attribute
 
 
-class EigerFanAdapterController(OdinAdapterController):
+class EigerFanAdapterController(OdinSubController):
     """Controller for an EigerFan adapter in an odin control server"""
 
+    async def initialise(self):
+        self._process_parameters()
```

Comment:
I would inline this into the for loop below and get rid of `_process_parameters`